docs/examples/good_practices/hpo_with_orion/README.rst

```diff
@@ -0,0 +1,333 @@
+Hyperparameter Optimization with Oríon
+======================================
+
+There are frameworks that allow to do hyperparameter optimization, like
+`wandb <https://wandb.ai/>`_,
+and `Oríon <https://orion.readthedocs.io/en/stable/index.html>`_.
+Here we provide an example for Oríon, the HPO framework developped at Mila.
+
+**Prerequisites**
+Make sure to read the following sections of the documentation before using this
+example:
+
+* :ref:`pytorch_setup`
+
+The full documentation for Oríon is available `on Oríon's ReadTheDocs page
+<https://orion.readthedocs.io/en/stable/index.html>`_.
+
+
+**job.sh**
+
+.. code:: diff
+
+    # distributed/001_single_gpu/job.sh -> good_practices/hpo_with_orion/job.sh
+   old mode 100644
+   new mode 100755
```

Comment:
this should probably not be there right?

Reply:
En effet, corrigé! https://github.com/mila-iqia/mila-docs/pull/198/commits/2a69570d1e6da409b4fe4993310aee1df1ef0305